### DISCUSSION TITLE
Uber Eats 推無人送餐模式，超可愛外送機器人不會被半路截餐嗎？

### DISCUSSION THREAD

**Discussion by Rosancl** [1101-GS4719-C/Python-Community-GS4719-C#609] [General]
針對自動送貨服務，Uber 正在加州兩個地區進行試點。在西好萊塢地區，Serve Robotics 和 Uber Eats 將使用人行道機器人提供短距離、非接觸式送貨服務；在聖莫尼卡地區，則會透過 Motional 製造的自動駕駛汽車為顧客送餐。

https://buzzorange.com/techorange/2022/05/17/uber-eats-robot/

這幾年真的有越來越多跨領域科技結合，讓生活更方便了～～
（自己先推推Serve Robotics 真的好可愛><）

**Comment by yucheng1028** [comment]
太可愛了八

**Comment by cordydeng** [comment]
在疫情肆虐的現在，對於減少人與人的交流，可說是非常有幫助的，如果未來能加上取餐時的身分驗證的話，應該就能避免被偷拿的問題

**Comment by henrywen547** [comment]
天哪好可愛呀
希望能更快推出長距離、快速且安全的服務

**Comment by jcpan821** [comment]
現在好像甚麼東西都要無人化了

**Comment by d26488399** [comment]
這樣外送員也可以免於等待客人來取餐，可以提高送餐效率

**Comment by 11timyyy** [comment]
若有發生突發狀況，例如被搶紅燈的車子撞到，不知道該如何解決 😅

**Comment by leon10456** [comment]
好可愛 但是在交通超亂的台灣應該沒辦法用 不然會一堆在地美食吧

**Comment by HsuChen88** [comment]
可能要有個像gogoro用app啟動車子的系統來限制只有訂的人可以打開這個蓋子，這樣食物就不會一直被搶走。

**Comment by sys511613** [comment]
這個出現之後 外送員們要集體失業啦

**Comment by Lai27330** [comment]
好可愛，但是台灣的路況真的很可怕，好像不太適合QQ

**Comment by alexthebigpp** [comment]
這種送餐機器人感覺也是很看一個地方人民的素質的，雖然說台灣人不太可能會去搶，但還有在運送過程產生的糾粉等等，因此要在台灣實施可能還要在一段時間。

**Comment by yenwei12138** [comment]
機器人很可愛沒錯，但這樣送餐速度就沒有像現在機車來的快，照片裡看起來像是跟人一樣走人行道，如果大幅拉長送餐速度就會不太想訂了

**Comment by Yao900108** [comment]
這個技術目前看來還有很多問題需要克服，但期待未來真的使用到!!

**Comment by stan890425** [comment]
在涮奶業也有可愛機器人送餐

**Comment by jcpan821** [comment]
在疫情下能有效降低接觸

**Comment by minniewei** [comment]
我覺得這是一個非常有用的東西

**Comment by Ben6uo** [comment]
如果飛高高的會不會突然故障掉下來?

**Comment by etin2468** [comment]
這在下雨天也可以送就能避免單子一直不被受理了

**Comment by 110409538** [comment]
好可愛~~

**Comment by Lynn12378** [comment]
超可愛!感覺之後可以很好解決現在外送員帶來的交通問題。

**Comment by Jimmy911218** [comment]
好可愛~而且感覺不會怕被他吃掉
